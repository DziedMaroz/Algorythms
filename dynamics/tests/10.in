4533602537929739021
632
8
6541806
093404960
608145
26121129646350
9207729
5852286087
467605218
961676373
76731207155499712
0889566
1546867973132
9775887610193
27358
5540253867286
083275
37
603775246
575383985796

334
276758644261291553
3
603190758386
6
931755728666633
3960923
00339884
1389594047393449
638904487178
88
012
8
4837968292
60256517027693
92650
505907479120926107
6585905211200030
85030100639183557
991927171
36685485380
17798360501223678
129350810999933965
14
191354771129759
27556458844104287
07853319
9661
4739
93118129498
8621583005
49
5
941534
992354858654236
685257380
5848371842749973
378911776921116773
241477707025640804
0903525447546138
1599038685204609
9
311055569
5253
98059833
445735
3624566

085445
77663989887
8
257504508
5

0929652270393
359573407783656597
902043
641760
892
9
88542171
154015385
1299971749008233
44554609920282539
1913628
63445345894
151348
484058380
244095991816503034
6762968
990646282656
3256042609
312348359889520300
1
52
852477726506499000
527070841904655523
6993952
027
149301
6
99772360510079
282733159340
6702214007220
04699987103
5277971
1220218913535
8705527
70673174853
0
18710466937651344
248
00498141
0985422402
3851
62784

020430170395078
7911862708066024
7
4022233948
4939506307734
72472062241
58327006993495
2314086497
54579010885
432523563764
8242193
82
4674267087756
118
802068814919863823
2027669
795024919591
8753203748748574
0380197264694
058160197
288407209984097528
0070761692752893
36020098054
751001001816767
3036638329
47
347801486362939977
95
0592

2
66694013058254453
094234679861827546
08721493998537

67171
3994272
02083466350
927282770709962
4668365560139661
735542447695211539
09992
94454437303085846
3734
251
043399269829423014
6125
8971668345914
947
3179748
55372208971162
51732826
0871362280548149
894881933421141
831583099733932326
582494090
743724677555
8036095147

1292
456811487105430
48034457591706
6923
84894376619
2407
777636579
83710019732
0096

6716
94240845234
7678
70531546213565998
1974797
4853142
8
4
015169
846
62986970095952659
52365036996395
89
8395
43569651
7281

865
094636742489773108
953
694855800989574
18588334992841118
64803473687435502
50473213083
837327370
637207100
58721
2061203756420
6105
66955165814550
14483341777752
9703
7280077996
656611453120149
803134840761

24
85148207890

846684861949467
73
6045020369549
32
862884643607931
6770480973535263
362485539553
7642196365085
9121795759450538
3580
08874937987036
784538794034
77
44503
24269680317
91745198
1037742992
6836638535484343

9944500
760546516684845
242
5807
7996940030908
95738746930
990996
6880501228
5056423734154
820291906937710
3195464824743497
804969466824794
2386815
9874487289
335980291579
421742
42
099
789620070940289
9
527018906024790491
35162978202
0
1912104137092549
555355892377
364032400630625595
30594
015150896879927
165
6844020610053348
7913026510996
0580067550

33718
68886
0423

93481

734773788642113
174
76438381768
13723835139790909
1252639806

212
5534646185
64659488
56474971143096199
07207
08337484

9882
44783
3308378
403279365247825
6758018399
600330810172
57391198693921
9613020
7832884
52
241767
533303
357833816984354
13921488333
581088904599
0683674672044

5593825114049107
11565493906703384
52941
50536084720839039
41060553192
432
771906
0200544953067351
638464842709607917
762670
25810139
0790381640
6203100461914264

309007
671439337636054

42441815612809527
95889763285
59163
71012364894880465
046145
9910237001177
30130713516157630

5593003576
2789
83260393076
62428
9283
26389502155213
602931003115
6963058126
287323
7854672
995
900986160459153
79291022117
6581
87
8516295707485144
6
83733525286844
22298465
18766931011

2
43498984
02543043

77522315437699
013
514
8142
714
6566
148277814761636
320
65662
343874060
501797039263444587
47231776802022973
1549016787140
02313421106106
883192398840
1087513908560
28960
540
92575
8296506929
9422934
800198708
698
16373893
4353
103476507956322414
5565
369619006693566394
35015553
1
0810644573
6783056
4128
04263
3029
55921174639376701
4306060395044302
740826785513055422
59821467695
264829286
1022547207
3
75727
17286287699
881027974641943793
2485
3194718
75394672891680143
405679808748513
979821121192
7472485984673
989644208699
220
47395496
56219732909
948441875839194
7750303
05361770014512
709015
14083784192251
68492302347916604
29702307637694
95
76906228
27004277224955269
2069
775930
8
38127755
74799088340
854149118877057
675997360917
3
315764889
49971051755
0379957
39216745792829762
2521
1990
4113527923
3
56173168685
14
463

74334014402
20059508
481436740225
93791652
111
2432220593
19899808571
890
66
5
4641542837263388
5090
011176

38074155890357623
3356918113
69332381655
2214686200466
935306971735695012
2537970
0839974773
99810524547922
17
96195631292546507
744374195574656620
293666576580336
60588799072837145
132
2387914721069150
55040775
5803898156867098

842
884838468630648
944
5445907969712
3060475991534841
81
1257726646494636
5588413468
563229933768
63103
337
568492614361675188
555960692908
755672
8492290824095426
84158546
248973628091059581

844059124742801
35728825141971633

165497859
62448
57974
77378370667231
3333796893
48608
817196490770
1337
149054001146078795
0557027575738309
403942311902
61869330745596
834252074482
7095689
2914732193661872
7167614129
807211410060286
015331174596
049083669
31750752
594318
52
14
58276427488
3
121380070332032
966647286
625680
0424
9954593
10281767021
529208015941
33335415199
82636
65
99426429923842
9073
02
8448
2748532134134639
117433740105850047
8039966606749
93919514778
49045769
33210425
46
25159
69564639073396
3477
746837
194710355194
158367612127454175
34008975865859
04943049218009291

0
92442247577151226
0166
3228203634
212008876134
4611
955489758
161
514078413
385141916
5
92352742
05
68
7941947
04
1824484009274

419405

6033987
9964465741507
46150398073631
725370616
17759
00
528096
359813421867865
2158986485107
35513742611
4930
494
1
480795
6
5310
0935134
92
6827554268
5809020
074909170096865670

72602098622217728
84712
239
50010640448919
65869
397386
31586494
7402351780601
3968870619609658
00441
99796373496875
08201826024
8560777937
932
42621044
703869679755
754723607
0435211070576904
9765745147406
6365783413
06098474907
9986383359
427610
9
78871618830521844
715361956402622
125532995781290740
9749448020
97548824332773
6453526892609
419031944

071089311076141
8371
74980816
