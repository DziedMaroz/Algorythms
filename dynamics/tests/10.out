14
4 5 3 3 6 02 5 37 92 9 73 9 02 1
